nixos/modules/programs/sway-beta.nix

```diff
@@ -0,0 +1,54 @@
+{ config, pkgs, lib, ... }:
+
+with lib;
+
+let
+  cfg = config.programs.sway-beta;
+  swayPackage = cfg.package;
+in {
+  options.programs.sway-beta = {
+    enable = mkEnableOption ''
+      Sway, the i3-compatible tiling Wayland compositor. This module will be removed after the final release of Sway 1.0"
```

Comment:
If you have time for it you might want to remove the last double quotes (...Sway 1.0"), as that might be visible in the manual, but we can also change that when merging ;)

Reply:
good eye, fixed. thanks.